### ISSUE TITLE
hollow circular wall with concentric fill

### ISSUE THREAD

**Issue by mrbi11** [alexrj/Slic3r#1654]
files are in: coneonly.stl, ini and .gco
https://app.sugarsync.com/iris/wf/D99423_83677863_6551580#cGFnZUlkPXdlYmxpbmtzJmlzV2VibGlua3NGb2xkZXI9dHJ1ZSZpc0l0ZW1SZWZyZXNoQWxsb3dlZD10cnVlJnVzZXJJZD0tMSZjdXJyZW50T3duZXJJZD00NDk3OCZjdXJyZW50Rm9sZGVySWQ9MzgxMjIzMThfMTAwNjAzNSZ3ZWJsaW5rSWQ9RDk5NDIzXzgzNjc3ODYzXzY1NTE1ODA=

Bug is similar to previous report in that it has some hollow walls.
Using concentric fill to try and get solid walls on circular sliced walls.
(hollow cone with top cut off).

alternate layers seem sliced solid, then the next layer will be solid on the
mostly horizontal parts, and hollow on mostly vertical parts.

fill set to concentric, so at a minimum i think each layer should be the same.
this is a tall cone, 12 inches or so.
pattern appears to persist all the way to the top.

If you have any suggested workarounds on how to get solid circular walls
in short term, thanks in advance.

PS. Note the link is to a different share provider.
Apparently google drive does not share to
"anyon on the web without them being logged in".
Hopefully, these are easier to access for anyone without gdrive.
please advise if that isn't working easily.

**Comment by lordofhyphens** [comment]
Sorry for the long wait, but the files you uploaded are gone now :(
You should be able to upload .zip files to github now, could you give it another try with 1.3.0-dev or 1.2.9 and see if it's still a problem?

Please include a screen capture of Slic3r's 3d preview. 
I'm going to close this, please reopen if it's still a problem.
